gammapy/modeling/models/cube.py

```diff
@@ -1093,6 +1093,50 @@ def stack(self, other, weights=None, nan_to_num=True):
         self.spectral_model.norm.value = 1
         self.spectral_model.tilt.value = 0
 
+    def slice_by_energy(self, energy_min=None, energy_max=None, name=None):
+        """Select and slice model template in energy range
+
+        Parameters
+        ----------
+        energy_min, energy_max : `~astropy.units.Quantity`
+            Energy bounds of the slice.
+        name : str
+            Name of the sliced model.
+
+        Returns
+        -------
+        model : `TemplateNpredModel`
+            Sliced Model
+
+        """
+        name = make_name(name)
+
+        energy_axis = self.map._geom.axes["energy"]
+
+        if energy_min is None:
+            energy_min = energy_axis.bounds[0]
+
+        if energy_max is None:
+            energy_max = energy_axis.bounds[1]
+
+        if name is None:
+            name = self.name
+
+        energy_min, energy_max = u.Quantity(energy_min), u.Quantity(energy_max)
+
+        group = energy_axis.group_table(edges=[energy_min, energy_max])
+
+        is_normal = group["bin_type"] == "normal   "
```

Comment:
"normal   ": this empty space is really what is expected??

Reply:
I copied that part of the code from datasets.slice_by_energy so I expect yes